bot/exts/filtering/filtering.py

```diff
@@ -1025,7 +1027,7 @@ async def _resolve_list_type_and_name(
                 await ctx.send(
                     "The **list_type** argument is unspecified. Please pick a value from the options below:",
                     view=ArgumentCompletionView(
-                        ctx, args, "list_type", [option.name for option in ListType], 0, list_type_converter
+                        ctx, args, "list_type", [option.name for option in ListType], 0, ListTypeConverter
```

Comment:
This isn't going to work, the class expects a function.

Reply:
Updated in https://github.com/python-discord/bot/pull/2523/commits/fae7cea3a948d170ca0fd5e5dfa6cf4e69054b5e